### ISSUE TITLE
Windows build contains a Linux ELF executable

### ISSUE THREAD

**Issue by mhewedy** [trivago/gollum#258]
```bash
$ curl -L -k https://github.com/trivago/gollum/releases/download/v0.5.4/gollum-0.5.4-Windows_x64.zip -o gollum-0.5.4-Windows_x64.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   618    0   618    0     0    753      0 --:--:-- --:--:-- --:--:--   753
100 5968k  100 5968k    0     0   242k      0  0:00:24  0:00:24 --:--:--  328k

mhewedy@N16497 /c/temp
$ unzip gollum-0.5.4-Windows_x64.zip
Archive:  gollum-0.5.4-Windows_x64.zip
  inflating: gollum

mhewedy@N16497 /c/temp
$ file gollum
gollum: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, stripped
```

**Comment by arnecls** [comment]
Found a problem in the makefile that caused this.
Fixed in 0.5.x, will pick it into 0.6.0, too.

Here is a correct build of the current 0.5.x branch.
[gollum-0.5.4-3-ce72d99d-Windows_x64.zip](https://github.com/trivago/gollum/files/3788449/gollum-0.5.4-3-ce72d99d-Windows_x64.zip)

**Comment by mhewedy** [comment]
Thanks.
